src/Components/Control.php

```diff
@@ -35,65 +35,40 @@
  */
 class Control extends Application\UI\Control
 {
-	/**
-	 * @persistent int
-	 */
-	public $page = 1;
+	public int $page = 1;
 
 	/**
 	 * Events
 	 *
-	 * @var array
+	 * @var mixed[]
 	 */
-	public $onShowPage;
+	public array $onShowPage;
 
-	/**
-	 * @var Utils\Paginator
-	 */
-	protected $paginator;
+	protected ?Paginator $paginator = NULL;
```

Comment:
Můžeme ještě všude upravit to `NULL` na malé písmena? To stejné jsem ještě viděl `TRUE`